### ISSUE TITLE
dotnet pack nuget package don't contains update time on file attributes

### ISSUE THREAD

**Issue by sdcb** [dotnet/cli#12678]
## Steps to reproduce
Just run: `dotnet pack` on a project folder.

## Expected  behavior
Dll file on nuget package should contains update time attribute.
![image](https://user-images.githubusercontent.com/1317141/65253001-79cc8000-db2c-11e9-8895-5f17b1274d87.png)

## Actual behavior
But it fixed to 01/01/1980
![image](https://user-images.githubusercontent.com/1317141/65252957-67eadd00-db2c-11e9-9f9a-431df324b910.png)

## Environment data
`dotnet --info` output:
```
 OS Name:     Windows
 OS Version:  10.0.18362
 OS Platform: Windows
 RID:         win10-x64
 Base Path:   C:\Program Files\dotnet\sdk\3.0.100-rc2-014266\

Host (useful for support):
  Version: 3.0.0
  Commit:  7d57652f33
```

# Note: 
* I'm experience this issue on just .NET Core 3.0, maybe since preview9(or earlier).
* When I using `dotnet pack` on .NET Core SDK 2.2, this is all good.
* If I create nuget package using msbuild.exe or by `nuget pack {projectFile} -build`, this is also all good.

**Comment by EdiWang** [comment]
I also met this problem. It causes my deployment tool (like Azure DevOps web deploy) not overriding server dll with new version of dll because it's modified time is empty, and eventually caused a .NET runtime exception for unable to load some assembly.

**Comment by livarcocc** [comment]
We have already addressed this problem. A fix for it will ship with the 3.0.100 GA SDK version.
